recipes/xpublish/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "xpublish" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 89d04cb0ff3a68c65ddf4c8f3400030e754b99edb0b4e615cad8f453b3435564
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3
```

Comment:
Since we are using f-strings, isn't it the case that we want `python >=3.6` here?